vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -330,6 +298,35 @@ void WinRTWebSocketResource::Synchronize() noexcept {
 #pragma region IWebSocketResource
 
 void WinRTWebSocketResource::Connect(string &&url, const Protocols &protocols, const Options &options) noexcept {
+  m_socket.MessageReceived(
+      [self = shared_from_this()](IWebSocket const &sender, IMessageWebSocketMessageReceivedEventArgs const &args) {
```

Comment:
>self [](http://example.com/codeflow?start=7&length=4)

(nit) I'm also curious whether weak_ptr would make more sense here. If object is destroyed, it's probably not worth processing any MessageReceived on it.